axelrod/tests/strategies/test_mutual.py

```diff
@@ -22,10 +22,37 @@ class TestDesperate(TestPlayer):
     def test_strategy(self):
         self.first_play_test(C, seed=1)
         self.first_play_test(D, seed=2)
-        self.responses_test([D], [C] * 4, [D] * 4)
-        self.responses_test([D], [D, D, C], [C, C, D])
-        self.responses_test([C], [D, D, D], [C, C, D])
 
+        #Our Player (Desperate) vs Cooperator SEED -->1
```

Comment:
Hello @MariosZoulias. I just have a few comments on PEP8. PEP8 is a style guide for python code, so it's just tips on making the code look nice and readable :smile: . There should a space after `#` and before the comment and I would add a space after `-->` as well. Thus, `# Our Player (Desperate) vs Cooperator SEED --> 1`. This applies for all the comments below.